modules/spring-data-2.0/pom.xml

```diff
@@ -67,6 +67,18 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
I don't like the fact that we force these dependencies on the users of ignite spring-data. Can we make those optional? Will it work? This way, users will have to depend on these libraries if they want the functionality.

Reply:
Mark as optional.